static/js/services/contact-schema.js

```diff
@@ -176,6 +176,16 @@ angular.module('inboxServices').service('ContactSchema',
         });
         return schema;
       },
+      getChildPlaceType: function(type) {
+        var schema = getSchema();
+        if (!_.has(schema, type)) {
+          return undefined;
```

Comment:
You can just return, I think it does the same thing. You could also flip this around to wrap the rest of the function in `if (_.has(schema, type))` and have the undefined return implicit.